log4cpp.sh

```diff
@@ -0,0 +1,35 @@
+package: log4cpp
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: REL_1_1_1_Nov_26_2013
+source: https://github.com/PMunkes/log4cpp
+build_requires:
+  - autotools
+---
+#!/bin/bash -ex
+rsync -a $SOURCEDIR/* .
+./autogen.sh
+./configure          --prefix=$INSTALLROOT  \
```

Comment:
You should align `\`. If you use `vim` I suggest you install the [tabular](https://github.com/godlygeek/tabular) plugin or something similar.